src/gmail.js

```diff
@@ -2529,10 +2529,10 @@ var Gmail = function(localJQuery) {
     var get_displayed_email_data_for_single_email = function(email_data) {
         var displayed_email_data = {};
         for (var id in email_data.threads) {
-            var message_class_id = "m"+id;
-            var displayed_email_element = $(".ii.gt .a3s.aXjCH." + message_class_id);
+            var message_class_id = id;
+            var displayed_email_element = document.querySelector("div[data-legacy-message-id='" + message_class_id + "']");
```

Comment:
This looks much better, but lets just drop the `message_class_id` variable all together:

* it's exactly the same as `id`
* it's not a message_class anymore

So yeah. Let's just drop it.

Reply:
If you can fixup your PR to only include this bit, with the tiny review-comment I had, that would be great and I will merge.